Dockerfiles/cluster-agent/amd64/Dockerfile

```diff
@@ -16,7 +16,7 @@ COPY readsecret.sh .
 
 RUN chmod 755 entrypoint.sh \
     && chown root:root readsecret.sh \
-    && chmod 555 readsecret.sh \
+    && chmod 510 readsecret.sh \
```

Comment:
The “execute only” bit on the shell script is useless because, on the contrary to an elf binary, in order to be executable, a script needs to be both executable and readable.

Demo:
```
vagrant@agent-dev-lenaic:/tmp$ ls -l hello hello.sh
-r-x--x--x 1 root root 16696 Aug 31 07:52 hello
-r-x--x--x 1 root root    42 Aug 31 07:51 hello.sh

# ELF binary
vagrant@agent-dev-lenaic:/tmp$ ./hello
Hello

# bash script
vagrant@agent-dev-lenaic:/tmp$ ./hello.sh
/bin/bash: ./hello.sh: Permission denied
```

This is because the executable bit is needed to start the script and the readable bit is needed for the shell interpreter to be allowed to read the script.

```suggestion
    && chmod 550 readsecret.sh \
```